src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -244,7 +244,7 @@ function getRefinementsItems({
   includedAttributes,
   excludedAttributes,
 }: {
-  results: SearchResults;
+  results: SearchResults | Record<string, never>;
```

Comment:
what is it for?

Reply:
I changed the signature of getRefinementsItem to that it can be called with an empty object too (like how it's done in a couple places, like query rules and current refinements before the results are in)